lib/matplotlib/collections.py

```diff
@@ -1795,6 +1795,42 @@ def _set_transforms(self):
             m[:2, 2:] = 0
             self.set_transform(_affine(m))
 
+    def _set_widths(self, widths):
+        self._widths = 0.5 * np.asarray(widths).ravel()
+
+    def _set_heights(self, heights):
+        self._heights = 0.5 * np.asarray(heights).ravel()
+
+    def _set_angles(self, angles):
+        self._angles = np.deg2rad(angles).ravel()
```

Comment:
I'm not sure if we really need these private helpers.

Reply:
I removed them.